ci/env.sh

```diff
@@ -0,0 +1,31 @@
+#smoke test environment specifications:
+export EACH_TEST_TIMEOUT=900s
+
+#for scale perf
+export ORIGINAL_WORKER_COUNT=`oc get nodes -l node-role.kubernetes.io/worker | grep -v NAME | wc -l`
+export NEW_WORKER_COUNT=$((ORIGINAL_WORKER_COUNT + 1))
+export SCALE=$NEW_WORKER_COUNT
+export RUNS=1
+
+#for kube burner 
+export JOB_ITERATIONS=3
+export CLEANUP=true
+export CLEANUP_WHEN_FINISH=true
+export NODE_COUNT=`oc get nodes -l node-role.kubernetes.io/worker | grep -v NAME | wc -l`
+export PODS_PER_NODE=40
+if [[ -n $KUBECONFIG_PATH ]]; then
```

Comment:
We might want to document this variable in the readme for the users to leverage it when using a kubeconfig which is not in the default location.

Reply:
yes makes sense , will add it right away